### PR TITLE
check arrays size

### PR DESCRIPTION
**Pull request by nirforer**
not sure if this is the solution here but I hope it helps :)

### CODE REVIEW

**Comment by nirforer** [comment]
not sure why it doesn't compile I will check this tomorrow. I see that the accessConfigs array is not on the get response when launching a vm without a public IP

**Comment by avivl** [comment]
The usage of KubeIP without public ip does not make any sense

**Comment by nirforer** [comment]
the below trace shows that the code hits such scenario on Geoedge cluster , this only started after the latest commit that accesses the accessConfigs array. Not sure when this happens (maybe between deleting the old nat and assigning the kube-ip)?

panic: runtime error: index out of range goroutine 1 [running]: github.com/doitintl/kubeip/pkg/kipcompute.replaceIP(0xc000173820, 0x11, 0xc0000bba10, 0xa, 0xc000340900, 0x2d, 0xc0000bb980, 0xc, 0xc0000c74a0, 0x0, ...) /go/src/github.com/doitintl/kubeip/pkg/kipcompute/compute.go:126 +0x12cc github.com/doitintl/kubeip/pkg/kipcompute.Kubeip(0xc0003be000, 0xc0000c74a0) /go/src/github.com/doitintl/kubeip/pkg/kipcompute/compute.go:208 +0x192 github.com/doitintl/kubeip/pkg/controller.Start(0xc0000c74a0, 0x0, 0x0) /go/src/github.com/doitintl/kubeip/pkg/controller/controller.go:119 +0x513 github.com/doitintl/kubeip/pkg/client.Run(...) /go/src/github.com/doitintl/kubeip/pkg/client/run.go:29 main.main() /go/src/github.com/doitintl/kubeip/main.go:54 +0x532

File: pkg/kipcompute/compute.go [hunk 1]
```diff
@@ -123,13 +123,15 @@ func replaceIP(projectID string, zone string, instance string, pool string, conf
 		logrus.WithFields(logrus.Fields{"pkg": "kubeip", "function": "replaceIP"}).Errorf("Instance not found %s zone %s: %q", instance, zone, err)
 		return err
 	}
-	accessConfigName := inst.NetworkInterfaces[0].AccessConfigs[0].Name
-	op, err := computeService.Instances.DeleteAccessConfig(projectID, zone, instance, accessConfigName, "nic0").Do()
-	if err != nil {
-		logrus.Errorf("DeleteAccessConfig %q", err)
-		return err
+	if len(inst.NetworkInterfaces) > 0 && len(inst.NetworkInterfaces[0].AccessConfigs) > 0 {
```
Comment (avivl) [inline pkg/kipcompute/compute.go]: A VM always have a network interface so the first check is redundant. 

Regardin accssConfig - from the docs:
`if there are no accessConfigs specified, then this instance will have no external internet access.`

We can leave this check however a VM without extenal internet access is not the use case for kubeip.

**Comment by avivl** [comment]
Code do not compile since you referring the op var here `op, err = computeService.Instances.AddAccessConfig(projectID, zone, instance, "nic0", accessConfig).Do()` and it was declare in a diffrent block `op, err := computeService.Instances.DeleteAccessConfig(projectID, zone, instance, accessConfigName, "nic0").Do()`

**Pull request merged**